aries-site/src/pages/index.js

```diff
@@ -57,10 +57,10 @@ const Index = () => {
       <Meta title={title} description={pageDetails.seoDescription} />
       <Box gap="large">
         <HomeTiles>
```

Comment:
Was wondering if we should change this to be a `row-responsive Box` or a `Grid` with flexible rows? That would prevent this huge gap in single column mode: 
![Screen Shot 2020-03-25 at 1 34 30 PM](https://user-images.githubusercontent.com/1756948/77578260-dd4db680-6e9d-11ea-9efd-1ee50dc67d4c.png)

Reply:
This problem is also in the current site Correct .?

Reply:
It is just due to the nature of the layout being created with Tiles that have consistent height, so the text content doesn't fill the height fully in this width -- Let me play with an idea I have and push a commit up

Reply:
Think I fixed it!

Reply:
Sweet! Thank you. I will preview as soon as the build completes.